cron/mapview_24h_clustered.sql

```diff
@@ -391,6 +414,41 @@ COMMIT TRANSACTION;
 
 
 
+-- also do the same for the devices
+BEGIN TRANSACTION;
+    UPDATE pre_outdev
+    SET xyt = (xyt::bit(64) & x'00000000003FFFFF')::int8
```

Comment:
When you write SIMD intrinsics and they remind you of SQL, then you go back to writing SQL.